### ISSUE TITLE
Logo branding on front page

### ISSUE THREAD

**Issue by ronissim** [calcom/cal.com#12781]
### Is your proposal related to a problem?

(Write your answer here.)

### Describe the solution you'd like

add the ability to customize background of website / other custom branding that includes our logo to the front booking page 

### Describe alternatives you've considered

(Write your answer here.)

### Additional context

(Write your answer here.)

### Requirement/Document

(Share it here.)

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by PeerRich** [comment]
we are working on more customisation options on a separate ticket

**Comment by ronissim** [comment]
Can you share where for me to add feedback there?
